### DISCUSSION TITLE
you exceeded your current quota,please check your plan and billling details

### DISCUSSION THREAD

**Discussion by pavison** [nextai-translator/nextai-translator#557] [Q&A]
您好，我更新了 API key 后仍旧报错：you exceeded your current quota,please check your plan and billling details，是因为新的一个月了吗还是 OpenAI 限制了免费 API quota呢，还是说需要 Loading 一会等明天看看？还望解答，谢谢！

**Comment by shamozhiwu** [comment]
遇到了同样的情况。不知道是怎么回事。

**Reply by WENSHAO521** [reply]
我建议你花钱，一个月5美金的限制

**Comment by WENSHAO521** [comment]
一般出现这个情况，就是免费的截止了，我的免费到4月1，很尴尬的一件事情

**Reply by shamozhiwu** [reply]
我看我的也说4月1日expire，我想知道单是这个月的用完了？还是说一次性的，以后彻底没了，都得花钱了啊？这个花钱买的究竟是什么？是API的使用权么？我用网页打开chatGPT，倒还是能用。我记得不是GPT-3.5是可以免费使用的嘛？有点没搞清楚。

**Reply by pavison** [reply]
你的网页版可以用吗？我的还可以，奇怪了。是不是说4月1日API截止啊

**Reply by shamozhiwu** [reply]
我的网页也可以用，只是API不行了，我也正在疑惑中。

**Comment by MikeLiu3030** [comment]
api选择的gpt4，发现不能用，请问如何更改到gpt3.5

**Reply by yetone** [reply]
点击左下角设置图标

<img width="42" alt="image" src="https://user-images.githubusercontent.com/1206493/229334420-747eaa22-98b0-4cec-8b69-7157b8e54ad9.png">

<img width="712" alt="image" src="https://user-images.githubusercontent.com/1206493/229334432-6a469f69-d75d-48f3-81f7-b3967271da2d.png">

**Comment by yetone** [comment]
<img width="599" alt="image" src="https://user-images.githubusercontent.com/1206493/229334479-31a0fda2-bb40-45b2-8866-d39a1cc0d94f.png">


https://twitter.com/yetone/status/1642076949219799041?s=20

**Comment by WENSHAO521** [comment]
一次性的，给18美金，有的人到6月，有的人到4月

shamozhiwu ***@***.***> 于 2023年4月2日周日 10:34写道：

**Comment by WENSHAO521** [comment]
chat gpt和api是分开的，chat gpt是免费的，api是一次性免费

shamozhiwu ***@***.***> 于 2023年4月2日周日 10:34写道：

**Comment by makai-china** [comment]
遇到类似问题，改成如下配置就好使了。 
![image](https://github.com/openai-translator/openai-translator/assets/11205189/f9c11a8a-95ad-4073-8f7d-24b35426f2ca)
